modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -4,11 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, examples
 
-
-The examples demonstrate common data extraction and transformation approaches.
+The following DataWeave examples demonstrate common data extraction and transformation approaches. There are general examples of how to transform data, and also examples of how to use DataWeave in Mule runtime engine (Mule) applications.
 //LINK TO DW 1.0 EXAMPLES:
 include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
 
+== DataWeave General Examples
```

Comment:
Not sure if people will understand what “DataWeave General Examples” means. I’m not sure what a good alternative is. Perhaps the page could avoid subsections, or maybe Kim or Cris have ideas here.

Reply:
DataWeave Code Examples ?

Reply:
See above ^

Reply:
I agree that "general" is not the right word. 
It seems to me that the division is between describing A Process and How A Process Is Implemented...maybe:

- DataWeave Code Examples?
- DataWeave Code Implementation (in Mule) Examples?